packages/design-system/src/components/contextMenu/contextMenu.js

```diff
@@ -17,29 +17,33 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { useMemo } from '@web-stories-wp/react';
 /**
  * Internal dependencies
  */
 import { Popover, Shadow } from './styled';
 import Menu, { MenuPropTypes } from './menu';
-import Mask from './mask';
+import AnimationContainer from './animationContainer';
+
+const ContextMenu = ({ animate, isAlwaysVisible, items, ...props }) => {
+  const Wrapper = useMemo(
+    () => (animate ? AnimationContainer : Popover),
```

Comment:
Animation is now determined by prop rather than needing to import the `AnimatedContextMenu`